library/build.gradle

```diff
@@ -21,6 +21,14 @@ android {
     defaultConfig {
         minSdkVersion 14
         targetSdkVersion 22
+
+        def projects = rootProject.getProject().getChildProjects()
+        projects.each {
+            def authority = it.value.android.defaultConfig.manifestPlaceholders.downloadAuthority
```

Comment:
@blundell do you even documentation?

http://gradle.org/docs/current/dsl/org.gradle.api.Project.html#org.gradle.api.Project:subprojects